cluster/gce/windows/k8s-node-setup.psm1

```diff
@@ -572,7 +573,8 @@ contexts:
   name: service-account-context
 current-context: service-account-context'.`
     replace('KUBEPROXY_TOKEN', ${kube_env}['KUBE_PROXY_TOKEN']).`
-    replace('CA_CERT', ${kube_env}['CA_CERT'])
+    replace('CA_CERT', ${kube_env}['CA_CERT']).`
+    replace('APISERVER_ADDRESS', ${apiserverAddress})
```

Comment:
apiserverAddress is not set here - this should be ${kube_env}['KUBERNETES_MASTER_NAME'] ?

Reply:
Ah you're right! Create-KubeletKubeconfig sets apiserverAddress, but not Create-KubeproxyKubeconfig!